src/templates/base/2019/contributors.html

```diff
@@ -353,6 +361,21 @@ <h1 class="title title-lg">
 {% block scripts %}
   {{ super() }}
   <script nonce="{{ csp_nonce() }}">
+  
+  // Randomise the contributors on reload as server-side random will be cached at CDN.
+  // Only run this if the page is reloaded for performance reasons.
+  
+  // Support both newer PerformanceNavigationTiming API for newer browsers
+  // and older Navigation Timing API for older browsers
+  if ((performance.getEntriesByType("navigation")[0] && performance.getEntriesByType("navigation")[0].type === "reload")
+      ||
+        (performance.navigation && performance.navigation.type === performance.navigation.TYPE_RELOAD)) {
+    var ul = document.querySelector('.contributor-grid');
+    for (var i = ul.children.length; i >= 1; i--) {
+        ul.appendChild(ul.children[Math.random() * i | 1]);
```

Comment:
Avoid magic
```suggestion
        ul.appendChild(ul.children[Math.floor(Math.random() * i)]);
```

Reply:
I need it to never return 0 so it never replaces the "Join the team" icon. What about this, if that makes it more obvious:

```suggestion
        ul.appendChild((ul.children[Math.random() * i) + 1]);
```

Will also add a comment.

Reply:
Ah ok. In that case is it possible to use a different collection of elements that doesn't include the one we don't want to shuffle?

Rather than `ul.children` maybe:
```
var contributors = document.querySelectorAll('li.contributor');
```

Reply:
Fixed.